frontend/src/app/patients/UploadPatients.tsx

```diff
@@ -281,14 +336,19 @@ const UploadPatients = ({ isAdmin }: { isAdmin: boolean }) => {
                             <FontAwesomeIcon icon={faXmark} />
                           </button>
                           <p className="usa-alert__text">
-                            {errorMessageText} See the{" "}
-                            <a
-                              target="_blank"
-                              href="/using-simplereport/manage-people-you-test/bulk-upload-patients/#preparing-your-spreadsheet-data"
-                            >
-                              patient bulk upload guide
-                            </a>{" "}
-                            for details about accepted values.
+                            {errorMessageText}
+                            {includeGuideInErrorMessage && (
+                              <div>
+                                See the{" "}
+                                <a
+                                  target="_blank"
+                                  href="/using-simplereport/manage-people-you-test/bulk-upload-patients/#preparing-your-spreadsheet-data"
+                                >
+                                  patient bulk upload guide
+                                </a>{" "}
+                                for details about accepted values.
+                              </div>
+                            )}
```

Comment:
A nice to have but non blocker would be taking the DOM of the error alert out of the general render and to a function

Reply:
Done 👍